app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -230,6 +230,22 @@ class QuestionPlayerActivityTest {
     }
   }
 
+  // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
+  @RunOn(TestPlatform.ESPRESSO) // Incorrectly passes on Robolectric and shouldn't be re-enabled
+  @Test
+  fun testQuestionPlayer_chooseCorrectAnswer_configurationChange_tickIsCompletelyVisible() {
```

Comment:
@MaskedCarrot Could you please mention why this can't be pass on robolectric? 
We need to fix the cause so that the test case should pass on both espresso and robolectric.

Reply:
all tests are correctly passing on roboelectric so I removed the TODOs now.